### PR TITLE
feat: Add ability to edit avatar

### LINKED ISSUE #11376: [CAL-2472] orgs: ability to edit avatar
CleanShot 2023-09-15 at 09 31 43@2x

<sub>[CAL-2472](https://linear.app/calcom/issue/CAL-2472/orgs-ability-to-edit-avatar)</sub>

### PR DESCRIPTION
**Pull request by samueldenzil**
## What does this PR do?

Added ability to edit the avatar.

Fixes #11376 

[screencast_2023-09-16_15-38-19.mov](https://github.com/calcom/cal.com/assets/71846487/2e1773a9-adc8-46d6-8775-c2d963b06dc0)

I've also added spacing of 4px between username and email.

Screenshot from 2023-09-16 15-41-59 1

### CODE REVIEW

File: packages/features/users/components/UserTable/EditSheet/EditUserForm.tsx [hunk 1]
```diff
@@ -88,12 +91,33 @@ export function EditForm({
...
+            <div className="flex items-center">
+              <Avatar alt={`${selectedUser?.name} avatar`} imageSrc={value} size="lg" />
+              <div className="ml-4">
+                <ImageUploader
+                  target="avatar"
+                  id="avatar-upload"
+                  buttonMsg="Change avatar"
+                  handleAvatarChange={(newAvatar) => {
+                    form.setValue("avatar", newAvatar, { shouldDirty: true });
+                    console.log(value);
```
Comment (Udit-takkar) [inline packages/features/users/components/UserTable/EditSheet/EditUserForm.tsx:113]: NIT: remove this

File: packages/features/users/components/UserTable/EditSheet/EditUserForm.tsx [hunk 2]
```diff
@@ -88,12 +91,33 @@ export function EditForm({
...
+          control={form.control}
+          name="avatar"
+          render={({ field: { value } }) => (
+            <div className="flex items-center">
+              <Avatar alt={`${selectedUser?.name} avatar`} imageSrc={value} size="lg" />
+              <div className="ml-4">
+                <ImageUploader
+                  target="avatar"
+                  id="avatar-upload"
+                  buttonMsg="Change avatar"
```
Comment (Udit-takkar) [inline packages/features/users/components/UserTable/EditSheet/EditUserForm.tsx:110]: use i18n here

**Review by Udit-takkar — CHANGES_REQUESTED** [review]
https://github.com/calcom/cal.com/assets/53316345/886d9b81-980f-4ef5-b0ca-9090c767427c

**Comment by samueldenzil** [comment]
> Screen.Recording.2023-09-22.at.2.40.02.PM.mov

This behaviour was happening before adding the avatar edit option. You must select the "role" first (for testing I was using the Admin role), then the update button works.

**Comment by Udit-takkar** [comment]
@samueldenzil  I don't this we should change the role of "OWNER", just to update the avatar.  i pushed some changes to fix the issue.

**Review by Udit-takkar — APPROVED** [review]
LGTM

**Pull request merged**